cli/run.go

```diff
@@ -71,6 +72,7 @@ func (rc *RunCommand) runRun(args []string) error {
 	}
 	containerName := rc.name
 	config.ContainerConfig.OpenStdin = rc.stdin
+	config.ContainerConfig.StdinOnce = rc.stdin // close stdin after attach connection closed
```

Comment:
if pouch run with -i  -d flag enable and when the terminal detach, this container can't be attached again, right?

Reply:
it has been blocked by 

```
if (rc.attach || rc.stdin) && rc.detach { 
return fmt.Errorf("Conflicting options: -a (or -i) and -d")
}
``

Reply:
> Maybe add a another flag for end-users to control is a good choice?

basically, I don't think so. The cli is just to help user to use pouch easier. This configuration is too complicated. I don't see any requirement for this.